scripts/peering-prefix

```diff
@@ -78,11 +85,33 @@ withdraw    Withdraw prefix to one or all muxes.
 -o asn      Prepend announcement to include the given ASN as the
             origin of the announcement.  Cannot be combined with -p;
             sets -P to 1 if not specified.  [default: unchanged (47065)]
--c id       Attach community (47065,id) to the announcement.  This
-            will let the announcement through the peer with the given
-            id.  May be added multiple times to announce to
-            multiple peers.  [default: announce to all peers]
-            (see https://peering.usc.edu/peers for the list of peers)
+
+-c id       Attach community (47065,id) to the announcement.  This will 
+            let the announcement through the peer with the given id
+            (id <= 1000) or specifically deny advertising to a peer
+            (if 1000 < id + 1000 <= 2000; e.g. to deny advertising to
+            peer 1: -c 1001).  May be added multiple times to announce
+            to multiple peers, but each peer will only see their
+            respective community (e.g.: -c 1 means Peer 1 receives
+            (47065,1)).  [default: announce to all peers only if no
+            communities attached] (see https://peering.usc.edu/peers
+            for the list of peers).  Can be combined with the -C.
+
+-C as1,c2   Attach community (id1,id2) to the announcement.  If the
```

Comment:
You call it `as1,c2` in the short hand, but then refer to them as `id1` and `id2` in the long description. In particular when you note "Only valid PEERING... are allowed for id1"